helm/action/create.go

```diff
@@ -1,33 +1,100 @@
 package action
 
 import (
+	"io/ioutil"
 	"os"
-	"path"
+	"path/filepath"
+	"text/template"
 )
 
 import (
 	"github.com/deis/helm/helm/log"
+	"github.com/deis/helm/helm/model"
 )
 
+// readmeSkel is the template for the README.md
+const readmeSkel = `# {{.Name}}
+
+Describe your chart here. Link to upstream repositories, Docker images or any
+external documentation.
+
+If your application requires any specific configuration like Secrets, you may
+include that information here.
+`
+
+// manifestSkel is an example manifest for a new chart
+const manifestSkel = `apiVersion: v1
+    kind: Pod
```

Comment:
since this is a raw string literal, we probably want this without the extra spacing at the start so it's all nicely formatted yaml.